### PR TITLE
Bugfix FXIOS-9335 Scaling Sync Button Correctly 

### PR DESCRIPTION
**Pull request by DanielDervishi**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-9335)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/20669)

## :bulb: Description

Made it so that the layout is updated after the sync setup view loads, allowing sync buttons to be resized before rendering

Before:

After:

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: firefox-ios/RustFxA/FirefoxAccountSignInViewController.swift [hunk 1]
```diff
@@ -238,7 +240,7 @@ class FirefoxAccountSignInViewController: UIViewController, Themeable {
         scanButton.applyTheme(theme: theme)
         emailButton.applyTheme(theme: theme)
 
-        scanButton.configuration?.image = signinSyncQRImage?.withRenderingMode(.alwaysTemplate)
+        scanButton.setImage(signinSyncQRImage?.withRenderingMode(.alwaysTemplate), for: .normal)
```
Comment (thatswinnie) [inline firefox-ios/RustFxA/FirefoxAccountSignInViewController.swift:243]: While this certainly works I don't think we should set the image this way. The button is using a button configuration and the image should be set the same way through the configuration. We should rather figure out why the button doesn't resize correctly when setting the image using the button configuration and fix that issue.

Comment (thatswinnie) [reply firefox-ios/RustFxA/FirefoxAccountSignInViewController.swift:243]: @DanielDervishi I think you forgot to change this code back to use the configuration.

Comment (DanielDervishi) [reply firefox-ios/RustFxA/FirefoxAccountSignInViewController.swift:243]: @thatswinnie Whoops, totally though I changed that back. Will fix ASAP

**Review by thatswinnie — APPROVED** [review]
Amazing! Good job finding the fix for this tricky UI bug 🎉

**Pull request merged**